view/template/acquisition/youtube.php

```diff
@@ -44,7 +44,7 @@
 
     <div class="content">
       <?php
-      if (isset($_GET['error'])): echo "<div>" . "The following error occurred: ". $_GET['error_message']  . " For support please send an email to hello@lbry.io" . "</div>";
+      if (isset($_GET['error']) && $_GET['error_message']): echo "<div>" . "The following error occurred: ". $_GET['error_message']  . " For support please send an email to hello@lbry.io" . "</div>";
```

Comment:
This check should also happen in the controller similar to the way I highlighted above:
```

  public static function executeXXX()
  {
    return ['acquisition/youtube', [
      'errorMessage' => isset($_GET['error']) && $_GET['error_message']) ? $_GET['error_message'] : null
    ]];
  }

Reply:
Above code could be improved via coalesce operator `??`